docs/install.rst

```diff
@@ -20,49 +20,55 @@ Taichi can be easily installed via ``pip``:
 Troubleshooting
 ---------------
 
-Taichi crashes with the following messages:
+CUDA issues
+***********
 
-.. code-block::
+- If Taichi crashes with the following messages:
 
-    [Taichi] mode=release
-    [Taichi] version 0.6.0, supported archs: [cpu, cuda, opengl], commit 14094f25, python 3.8.2
-    [W 05/14/20 10:46:49.549] [cuda_driver.h:call_with_warning@60] CUDA Error CUDA_ERROR_INVALID_DEVICE: invalid device ordinal while calling mem_advise (cuMemAdvise)
-    [E 05/14/20 10:46:49.911] Received signal 7 (Bus error)
+    .. code-block::
 
+        [Taichi] mode=release
+        [Taichi] version 0.6.0, supported archs: [cpu, cuda, opengl], commit 14094f25, python 3.8.2
+        [W 05/14/20 10:46:49.549] [cuda_driver.h:call_with_warning@60] CUDA Error CUDA_ERROR_INVALID_DEVICE: invalid device ordinal while calling mem_advise (cuMemAdvise)
+        [E 05/14/20 10:46:49.911] Received signal 7 (Bus error)
 
-This may because your NVIDIA card is pre-Pascal and therefore does not support `Unified Memory <https://www.nextplatform.com/2019/01/24/unified-memory-the-final-piece-of-the-gpu-programming-puzzle/>`_.
 
-* Try adding ``export TI_USE_UNIFIED_MEMORY=0`` to your ``~/.bashrc``. This disables unified memory usage in CUDA backend.
+  This may be because your NVIDIA card is pre-Pascal and does not support `Unified Memory <https://www.nextplatform.com/2019/01/24/unified-memory-the-final-piece-of-the-gpu-programming-puzzle/>`_.
```

Comment:
I'd say ''This might be due to the fact that ...''

Reply:
BTW, unified memory is not supported since GTX770? Maybe I remembered a lot of things wrong about CUDA...
So if I disable the unified memory, what computation capability do I need to use Taichi?

Reply:
> BTW, unified memory is not supported since GTX770? Maybe I remembered a lot of things wrong about CUDA...

NVIDIA has a fallback: https://devblogs.nvidia.com/unified-memory-cuda-beginners/ see `What Happens on Kepler When I call cudaMallocManaged()?` These GPUs doesn't support `memadvise` though...



> So if I disable the unified memory, what computation capability do I need to use Taichi?

Taichi doesn't have a hard requirement for that, but GTX 9 series should work (without unified memory/adaptive memory pool allocation)...I haven't tested GTX 7 series (which release 7 years ago).

Reply:
Oh thanks for that. I had a GTX770 for a long time and tried the unified memory ever since it was available (I think it was CUDA 6.0 but I'm not sure). I found it did a lot of unnecessary memory synchronization and I couldn't turn it off, so I just decided never use it again since then 😢